### PR TITLE
Make dev configuration customizable

### PR DESCRIPTION
**Pull request by Gounlaf**
`.gitignore` was already ignoring dev.custom.php (https://github.com/Wolnosciowiec/file-repository/blob/master/.gitignore#L5)

### CODE REVIEW

File: config/dev.php [hunk 1]
```diff
@@ -17,6 +23,10 @@
 $app['debug'] = true;
 $app['weburl'] = 'http://localhost:8888';
```
Comment (blackandred) [inline config/dev.php]: You left this :smile:

Comment (Gounlaf) [inline config/dev.php]: Yup, 'cause I've already removed it in the other PR!

Comment (Gounlaf) [inline config/dev.php]: Oh, sorry, I will fix it "again", 'cause there will be conflict ><

Comment (Gounlaf) [inline config/dev.php]: Done :)

**Pull request merged**